src/ffi/mod.rs

```diff
@@ -1,25 +1,3 @@
-//! Raw FFI declarations for Python's C API.
-//!
-//! This module provides low level bindings to the Python interpreter.
-//! It is meant for advanced users only - regular PyO3 users shouldn't
-//! need to interact with this module at all.
-//!
-//! The contents of this module are not documented here, as it would entail
-//! basically copying the documentation from CPython. Consult the [Python/C API Reference
-//! Manual][capi] for up-to-date documentation.
-//!
-//! # Safety
-//!
-//! The functions in this module lack individual safety documentation, but
-//! generally the following apply:
-//! - Pointer arguments have to point to a valid Python object of the correct type,
-//! although null pointers are sometimes valid input.
-//! - The vast majority can only be used safely while the GIL is held.
-//! - Some functions have additional safety requirements, consult the
-//! [Python/C API Reference Manual][capi]
-//! for more information.
-//!
-//! [capi]: https://docs.python.org/3/c-api/index.html
 #![allow(
```

Comment:
These allows should probably go into pyo3-ffi's lib.rs.

Reply:
I have addresses these attributes in both the lib.rs and mod.rs file, because the datetime.rs file (which is still inside pyo3 itself) still requires these allows. I could move these to the `datetime.rs` file however.

Reply:
Now that the datetime API has been removed from the pyo3 ffi module I was able to remove these attributes aswell